src/components/MyAccount/IconListElement.tsx

```diff
@@ -0,0 +1,52 @@
+import { Icon } from "@nypl/design-system-react-components"
+import type { IconNames } from "@nypl/design-system-react-components"
+
+import styles from "../../../styles/components/MyAccount.module.scss"
+
+interface IconListElementPropType {
+  icon: IconNames
+  term: string
+  description: string
+}
+
+const IconListElement = ({
+  icon,
+  term,
+  description,
+}: IconListElementPropType) => {
+  const removeTrailingColon = (term: string) => {
```

Comment:
Another option is to remove the colon from the defined string and add the colon on line 28 when the string is rendered, but nbd,

Reply:
i hadnt pushed up the changes that actually removed this logic entirely.